### PR TITLE
test/gh1247 [RCS-292]

### PR DESCRIPTION
**Pull request by kuenishi**
- Use batch_size preset when starting riak_cs_gc_batch process
- Add QuickCheck test to address #1247 
- Add console option (`-b`) to `riak-cs-admin gc batch` to manually set batch size in each batch

### CODE REVIEW

File: src/riak_cs_gc_batch.erl [hunk 1]
```diff
@@ -101,8 +102,8 @@ init([#gc_batch_state{
             %% EndKey <= BatchStart - Leeway
             _ = lager:info("Starting garbage collection: "
                            "(start, end) = (~p, ~p), "
-                           "leeway=~p, batch_start=~p, max_workers=~p",
-                           [StartKey, EndKey, Leeway, BatchStart, MaxWorkers]),
+                           "leeway=~p, batch_start=~p, max_workers=~p, batch_size=~p",
```
Comment (shino) [inline src/riak_cs_gc_batch.erl]: Log messages is a kind of end user / operator interface, too. It's better to use the same word here as command line option.

**Comment by kuenishi** [comment]
I've force-pushed, again, sorry :imp:

**Comment by shino** [comment]
The EQC prop actually catched the bug #1247  :cool:

**Comment by shino** [comment]
@borshop merge

**Pull request merged**